tests/MongoDB.Driver.Examples/ChangeStreamExamples.cs

```diff
@@ -122,5 +126,90 @@ public void ChangeStreamExample3()
                 next.FullDocument.Should().Be(documents[1]);
             }
         }
+
+        [Fact]
+        public void Changestream_Example_4()
+        {
+            RequireServer.Check().Supports(Feature.AggregateAddFields);
+
+            var client = DriverTestConfiguration.Client;
+            var database = client.GetDatabase("ChangeStreamExamples");
+            database.DropCollection("inventory");
+            var inventoryCollection = database.GetCollection<BsonDocument>("inventory");
+
+            //const pipeline = [
+            //{ $match: { $or : [{ 'fullDocument.username': 'alice' }, { 'operationType' : 'delete' }] } },
+
+            //{ $addFields: { newField: 'this is an added field!' } }
+
+            //];
+            //const collection = db.collection('inventory');
+            //const changeStream = collection.watch(pipeline);
+            //const next = await changeStream.next();
+
+            var cancelationTokenSource = new CancellationTokenSource();
+            try
+            {
+                BsonDocument document = new BsonDocument("username", "alice");
+
+                Task.Run(() =>
+                {
+                    while (!cancelationTokenSource.IsCancellationRequested)
+                    {
+                        Thread.Sleep(TimeSpan.FromMilliseconds(500));
+                        inventoryCollection.InsertOne(document);
+                        inventoryCollection.DeleteOne(document);
+                    }
+                }, cancelationTokenSource.Token);
+
+                // Start Changestream Example 4
+                var pipeline = new EmptyPipelineDefinition<ChangeStreamDocument<BsonDocument>>()
+                    .Match(filter =>
+                        filter.FullDocument["username"] == "alice" ||
+                        filter.OperationType == ChangeStreamOperationType.Delete)
+                    .AppendStage(
```

Comment:
Maybe add a comment like:

     //  Currently no helper method for $addFields

Otherwise it looks weird that we use Match followed by the generic AppendStage

Reply:
I will add this comment, but it looks like it's better to add it outside of the example delimiter brackets